docs/testing/troubleshooting.md

```diff
@@ -31,9 +31,9 @@ When your Excel Script encounters a problem running, it produces an error. You'l
 The following steps should help troubleshoot any problems related to the **Automate** tab not appearing in Excel on the web.
 
 1. [Make sure your Microsoft 365 license includes Office Scripts](../overview/excel.md#requirements).
-1. [Have your admin enable the feature](/microsoft-365/admin/manage/manage-office-scripts-settings).
```

Comment:
We should still mention here that their admin has to have the feature enabled. This could be one of the reasons why we show the error page (admins disabled it)

Reply:
This is called out in the new bulletpoint below. I wanted to frame it in the light that admin disabled the feature, instead of not enabling it.

Reply:
yeah sounds good